examples/src/croncat.rs

```diff
@@ -105,7 +105,7 @@ pub async fn run_scheduled_tasks(
     let outcome = agent
         .call(contract.id(), "register_agent")
         .args_json(json!({}))
-        .deposit(parse_near!("0.00226 N"))
+        .deposit(NearToken::from_yoctonear(2260000000000000000000u128))
```

Comment:
Sounds like we potentially need `from_micronear()` constructor in near-token, or maybe just use `new_or_panic()` in such cases and hope that the compiler will be able to resolve it at compile time.